### DISCUSSION TITLE
Serializable DTOs

### DISCUSSION THREAD

**Discussion by runningjustin** [babyfish-ct/jimmer#495] [Q&A]
Hey - Jimmer is fantastic! Just one problem at the moment - I am generating DTOs based on my model but I can't use them in ktor routes because they are not serializable. The examples which use @PostMapping do not help as I'm not using Spring Boot.

Am I missing something or is this just something I'm unable to do at the moment? If the latter, could this be added?

**Comment by babyfish-ct** [comment]
Do you mean the annotation `@Serializable`? DTO language supports user annotation for both top level DTO and nested DTO.

**Reply by runningjustin** [reply]
Aha yes, and it does indeed work. Thank you!

**Comment by babyfish-ct** [comment]
That's my fault, I used discord to create invitation link at first time a few weeks ago but I did not see that link can only be used for 7 days.

The never-expiration invitation link is https://discord.gg/PmgR5mpY3E, and I will update the like in the documentation in a few days.
